scanner/src/main/kotlin/scanners/fossid/Utils.kt

```diff
@@ -28,7 +28,7 @@ import org.ossreviewtoolkit.model.config.PathExclude
 
 private val DIRECTORY_REGEX = "(?<directory>(?:\\w+/)*\\w+)/?(?<starstar>\\*\\*)?".toRegex()
 private val EXTENSION_REGEX = "\\*\\.(?<extension>\\w+)".toRegex()
-private val FILE_REGEX = "(?<file>[\\w.]+)".toRegex()
+private val FILE_REGEX = "(?<file>[\\w.\\-]+)".toRegex()
```

Comment:
Could this be simplified to just not allow slashed, in which case the path exclude pattern should always point to a file? Like `"(?<file>[^/]+)".toRegex()`.

Reply:
I'm specifically asking was there are more non-word characters that are valid in file names, so maintaining this as an "allow-list" becomes cumbersome.

Reply:
Yeah I thought about it. For instance if somebody has Unicode characters...
Fixed.

Reply:
BTW, you should probably do something similar for `EXTENSION_REGEX`. Also, for a path exclude pattern like "*.tar.gz", what is `EXTENSION_REGEX` supposed to match / return? Really "tar.gz", or maybe just "gz"?